doc/classes/ImmediateGeometry.xml

```diff
@@ -66,7 +67,7 @@
 			<argument index="0" name="color" type="Color">
 			</argument>
 			<description>
-				Set the color that the next vertex will use to be drawn.
+				Sets the color that the next vertex will use to be drawn.
```

Comment:
Ref writing guidelines: "Don’t repeat the method’s name. It already states it sets":

  The current drawing color.